jraph/_src/utils.py

```diff
@@ -1122,3 +1123,33 @@ def wrapper(graph: gn_graph.GraphsTuple) -> gn_graph.GraphsTuple:
     return zero_out_padding(graph_net(graph))
 
   return wrapper
+
+
+def from_scipy_sparse_matrix(A: scipy.sparse.spmatrix):
+  """Creates a `jraph.GraphsTuple` from a scipy sparse matrix.
+  
+  Args:
+    A: A SciPy sparse matrix.
+    
+  Returns:
+    A `jraph.GraphsTuple` graph based on the scipy sparse matrix `A`.
+  """
+  A = A.tocoo()
+  n_node = jnp.asarray([A.shape[1]])
```

Comment:
could we stick to numpy arrays here and below.

Reply:
I'm sorry but I'm a bit confused on where should we use numpy arrays and where should we use jax.numpy arrays in utilities? I've seen other code in utilities using a bit of both so I'm not entirely sure.